static/templates/user_info_popover_content.hbs

```diff
@@ -150,4 +150,23 @@
             <i class="fa fa-paper-plane" aria-hidden="true"></i> {{#tr this}}View messages sent{{/tr}}
         </a>
     </li>
+
+    {{#if can_mute }}
+    <hr>
+    <li>
+        <a href="#" class="sidebar-popover-mute-user">
```

Comment:
I think we want this link to not have an `href`, rather than having `"#"` as the href, after looking at other similar links in this file.  (Fixing this will prevent the browser from showing a URL when hovering this link, which is correct as no URL corresponds to this action).

Reply:
Fixed.